### DISCUSSION TITLE
Input 10x multiome data alternative? (bypassing cellranger-arc aggr)

### DISCUSSION THREAD

**Discussion by dru93** [stuart-lab/signac#1197] [Q&A] [answered]
Hi, I have a set of samples sequenced with the 10x multiome kit.
Following the vignette of "Joint RNA and ATAC analysis: 10x multiomic" in order to create a Seurat object with my data I input the raw data via the filtered_feature_bc_matrix.h5 and atac_fragments.tsv.gz files that were previously generated by cellranger-arc aggr.
I have a lot of samples and cellranger-arc aggr takes considerable time to complete - days, so I'd really like to remove the cellranger-arc aggr step.
Is there a way of creating the same Seurat object by inputting the outputs of cellranger-arc count instead of the output of cellranger-arc aggr?

**Comment by timoast** [comment, marked as answer]
Yes, you can follow the merge workflow shown here: https://satijalab.org/signac/articles/merging.html
